### DISCUSSION TITLE
No Extra features in feature table

### DISCUSSION THREAD

**Discussion by pm404** [mzmine/mzmine#2060] [Q&A]
Hi, I'm new to mzmine. I'm trying to do molecular networking with my samples. I've processed my data and now I'm looking at the alignment. Does anybody know what it means if my samples have 0 summed extra features?

![image](https://github.com/user-attachments/assets/2e286660-6f4c-4b44-98d2-807138c8c4f7)

**Comment by ansgarkorf** [comment]
Hi,
please have a look at the documentation:
https://mzmine.github.io/mzmine_documentation/module_docs/align_join_aligner/join_aligner.html
<img width="758" alt="image" src="https://github.com/user-attachments/assets/c368108c-f86c-4830-80bd-2bce95a913eb">

**Reply by pm404** [reply]
Hi, I've checked this, thanks! I knew that if I had extra features, I might have isomers in my samples. But if there are none, should it be a concern?
